### DISCUSSION TITLE
"Fix" should not remove scripts

### DISCUSSION THREAD

**Discussion by scottaohara** [sinedied/a11y-ai#11] [General]
If the intent of this tool is to help create meaningful fixes that could be accepted / merged into code repositories, then the fix command should not be removing scripts it encounters.  While many developers would likely notice this and revert these specific changes when they were reviewing the diff of the changes the fix command made, it's an extra step that has the potential to break pages, rather than fix them, if a removal goes unnoticed.

**Comment by sinedied** [comment]
Oops, this is definitely a bug that was introduced when preprocessing was added, as it was not intentional.
The tool should only remove scripts for generating the fixes, but then a diff with the changes it generated against the version that was sent (without the scripts), so this diff is applied to the original files to get the scripts intacts.
I'll create an issue, thanks for the notice!
